nbs/02_match_class.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.            elif pattern_dst in self.collection_mapping:    
</pre><p>there is too much shared logic in your cases.</p><p>if source and target are not collections simply make a set with one value and then use it when doing product.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>